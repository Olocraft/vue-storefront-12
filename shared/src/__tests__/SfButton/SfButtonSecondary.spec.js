import { shallowMount } from '@vue/test-utils'
import SfButtonSecondary from '../../components/SfButton/SfButtonSecondary.vue'
let wrapper

describe('SfButtonSecondary.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(SfButtonSecondary)
  })
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
