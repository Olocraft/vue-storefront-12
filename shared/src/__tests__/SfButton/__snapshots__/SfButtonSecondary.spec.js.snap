// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SfButtonSecondary.vue render component correctly 1`] = `<button class="inline-flex items-center justify-center cursor-pointer border border-green-400 disabled:border-0 bg-white hover:bg-green-100 active:bg-green-200 disabled:bg-gray-200 text-green-500 hover:text-green-500 active:text-green-600 disabled:text-gray-500 disabled:opacity-50 uppercase px-[16px] py-2 rounded-md font-body text-base font-medium shadow hover:shadow-md disabled:shadow-none disabled:cursor-not-allowed"></button>`;
